Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mhart/alpine-node:6
+MAINTAINER code.vire@gmail.com
+
+RUN npm install -g yarn
+
+COPY package.json /tmp
+RUN cd /tmp &&\
+    yarn &&\
+    mkdir /app &&\
+    mv node_modules /app/node_modules &&\
+    rm package.json
+
+WORKDIR /app
+COPY . /app
```

Comment:
I'm not sure about your exact use case but your are right with catching but as it's only local catching I don't see it as much valuable but YMMV.

Also any reason to not use yarn.lock file to ensure exact packages versions?